tests/autologging/fixtures.py

```diff
@@ -31,3 +49,30 @@ def test_mode_on():
             os.environ[_AUTOLOGGING_TEST_MODE_ENV_VAR] = prev_env_var_value
         else:
             del os.environ[_AUTOLOGGING_TEST_MODE_ENV_VAR]
+
+
+@pytest.fixture(autouse=True)
+def reset_stderr():
+    prev_stderr = sys.stderr
+    yield
+    sys.stderr = prev_stderr
+
+
+@pytest.fixture(autouse=True)
```

Comment:
Since some of our tests disable MLflow event logging to test silent mode, this fixture is used to re-enable MLflow event logging at the end of each test.